mne/io/tests/test_apply_function.py

```diff
@@ -0,0 +1,35 @@
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import os.path as op
+from nose.tools import assert_equal
+
+from mne import create_info
+from mne.io import RawArray
+from mne.utils import logger, set_log_file, _TempDir
+
+
+def test_apply_function_verbose():
+    """Test apply function verbosity
+    """
+    def printer(x):
+        logger.info('exec')
```

Comment:
Probably safer to check to see if the return value is an array of the proper shape, I'll add a check. If someone *really* wants `nan`, they should have to return an array of `nan` instead of `None`.